app/Observers/UserObserver.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * NOTICE OF LICENSE.
+ *
+ * UNIT3D is open-sourced software licensed under the GNU Affero General Public License v3.0
+ * The details is bundled with this project in the file LICENSE.txt.
+ *
+ * @project    UNIT3D
+ *
+ * @license    https://www.gnu.org/licenses/agpl-3.0.en.html/ GNU Affero General Public License v3.0
+ * @author     HDVinnie
+ */
+
+namespace App\Observers;
+
+use App\Models\User;
+use Illuminate\Support\Facades\Cache;
+
+class UserObserver
+{
+    /**
+     * Handle the User "created" event.
+     *
+     * @param \App\Models\User $user
+     *
+     * @return void
+     */
+    public function created(User $user)
+    {
+        Cache::put("user.{$user->passkey}", $user);
+    }
+
+    /**
+     * Handle the User "retrieved" event.
+     *
+     * @param \App\Models\User $user
+     *
+     * @return void
+     */
+    public function retrieved(User $user)
+    {
+        Cache::add("user.{$user->passkey}", $user);
```

Comment:
Same as previous comment; why not use `put()` here?

Reply:
Good point.